sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuJsonToStructs.scala

```diff
@@ -172,15 +173,23 @@ case class GpuJsonToStructs(
         // Step 1: verify and preprocess the data to clean it up and normalize a few things
         // Step 2: Concat the data into a single buffer
         val (isNullOrEmpty, combined) = cleanAndConcat(input.getBase)
+
         withResource(isNullOrEmpty) { isNullOrEmpty =>
           // Step 3: setup a datasource
           val table = withResource(new JsonDeviceDataSource(combined)) { ds =>
             // Step 4: Have cudf parse the JSON data
-            cudf.Table.readJSON(cudfSchema, jsonOptions, ds)
+            try {
+              cudf.Table.readJSON(cudfSchema, jsonOptions, ds)
+            }catch{
```

Comment:
nit: The style guide indicates that there should be spaces in here

```
} catch {
```

Reply:
Thanks for the comments.